shipyard_blueprints/postgresql/shipyard_postgresql/cli/execute_sql.py

```diff
@@ -39,53 +28,34 @@ def get_args():
     return args
 
 
-def create_connection_string(args):
-    """
-    Set the database connection string as an environment variable using the keyword arguments provided.
-    This will override system defaults.
-    """
-    if args.db_connection_url:
-        os.environ["DB_CONNECTION_URL"] = args.db_connection_url
-    elif args.host and args.database:
-        os.environ["DB_CONNECTION_URL"] = (
-            f"postgresql://{args.username}:{args.password}@{args.host}:{args.port}/{args.database}?{args.url_parameters}"
-        )
-
-    db_string = os.environ.get("DB_CONNECTION_URL")
-    return db_string
-
-
-def create_db_connection(db_string):
-    if "db.bit.io" in db_string:
-        db_connection = create_engine(
-            db_string, connect_args={"sslmode": "require"}, isolation_level="AUTOCOMMIT"
-        )
-    else:
-        db_connection = create_engine(db_string)
-    Session = sessionmaker(bind=db_connection)
-    return db_connection, Session
-
-
 def main():
     args = get_args()
     query = text(args.query)
-
-    db_string = create_connection_string(args)
+    client_args = {
```

Comment:
Sorry if I wasn't clear. postgres = None should be the only thing outside of the try block.

end goal is to avoid any risk of a `NameError` in the finally clause. 

```
    client_args = {
        "user": args.username,
        "pwd": args.password,
        "host": args.host,
        "database": args.database,
        "port": args.port,
        "url_params": args.url_parameters if args.url_parameters != "" else None,
    }
    postgres = PostgresClient(**client_args)
    ```
    has a chance of failing and if it fails outside of the try block it will throw a traceback error